agent_workspace/smarthome-ai/.env.example

```
@@ -0,0 +1,34 @@
+NODE_ENV=production
+REACT_APP_API_BASE_URL=https://api.smarthome-ai.com
+REACT_APP_VERSION=1.0.0
+
+# 联系表单配置（第三方服务）
+# 选项 1: Formspree
+REACT_APP_FORMSPREE_ENDPOINT=https://formspree.io/f/your-form-id
+
+# 选项 2: EmailJS
+REACT_APP_EMAILJS_SERVICE_ID=your-service-id
+REACT_APP_EMAILJS_TEMPLATE_ID=your-template-id
+REACT_APP_EMAILJS_PUBLIC_KEY=your-public-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_EMAILJS_PUBLIC_KEY key should go before the REACT_APP_EMAILJS_SERVICE_ID key

(UnorderedKey)